### PR TITLE
#160617654 Liking and disliking a comment feature

### PR DESCRIPTION
**Pull request by kimbugp**
#### What does this PR do?
- allow a logged in user to like or unlike a comment 
#### Description of Task to be completed?
- Click on the like or dislike icon on the comments section.
#### How should this be manually tested?
- Go to this [URL](https://ah-frontend-lannister-pr-29.herokuapp.com/)
- Login, then view and a single article.
#### What are the relevant pivotal tracker stories?
- [#160617654](https://www.pivotaltracker.com/story/show/160617654)
#### Screenshots 

screen shot 2018-11-22 at 16 00 24

### CODE REVIEW

**Comment by LehruAngela** [comment]
Look at how @AmosWels implemented his actions(view more, delete )on the drafts/published page, your replies button would look better that way. Please change them.

**Comment by nakiwuge** [comment]
@LehruAngela we have made the changes

File: src/views/commentsViews/commentsCard.js [hunk 1]
```diff
@@ -18,9 +18,15 @@ import Moment from "react-moment";
 const CommentCard = props => (
   <div className="comment">
     <ViewCard {...props} />
-    <Button outline color="primary" id={`reply${props.id}`}>
+    <Button  color="primary" id={`reply${props.id}`}className="replies-btn">
```
Comment (LehruAngela) [inline src/views/commentsViews/commentsCard.js]: This Replies button could also look similar to the Likes button, that is, without the borders of the button, like a mere text, Please make that change to the implementation. @zellymiriam @kimbugp

Comment (nakiwuge) [reply src/views/commentsViews/commentsCard.js]: @LehruAngel we have changed it to text.

**Pull request merged**